src/tag-attr-list.js

```diff
@@ -239,4 +239,8 @@ Whitelist.HrefAttributes = [
     "xlink:href"
 ];
 
+// Void elements only have a start tag; end tags must not be specified for void elements.
+// https://html.spec.whatwg.org/multipage/syntax.html#void-elements
+Whitelist.VoidElements = {"area":1, "base":1, "br":1, "col":1, "embed":1, "hr":1, "img":1, "input":1, "keygen":1, "link":1, "menuitem":1, "meta":1, "param":1, "source":1, "track":1, "wbr":1};
```

Comment:
object lookup is generally faster than Array.indexOf(). 
but the actual performance difference could depend on the JS engine (nodejs, Chrome, or IE...)